bootstrap/api/v1beta2/ck8sconfig_types.go

```diff
@@ -124,6 +125,42 @@ type CK8sInitConfiguration struct {
 	EnableDefaultNetwork *bool `json:"enableDefaultNetwork,omitempty"`
 }
 
+// GetEnableDefaultNetwork returns the EnableDefaultNetwork field.
+// If the field is not set, it returns true.
+func (c *CK8sInitConfiguration) GetEnableDefaultDNS() *bool {
```

Comment:
Make this return a `bool`. Either the value of the field, or the default value (true).

Same for the others

Reply:
In this case, should we change the `UserFacingConfig` too? These values are `*bool`

Reply:
no, the acessors will return a `bool`, because they will always return a bool value, and we want callers to not have to worry about `*bool` being nil or not.

the API stays as is, and we simply add a `ptr.To()` when creating the config file.